src/views/AmalgamationSelection.vue

```diff
@@ -209,6 +211,39 @@ export default class AmalgamationSelection extends Vue {
     }
   }
 
+  /** Called when Start Short form Horizontal button is clicked. */
+  async startHorizontalAmalgamation (): Promise<any> {
+    // Create a draft amalgamation application then redirect to Create UI.
+    try {
+      // show spinner since this is a network call
+      this.setStartingAmalgamationSpinner(true)
+      const businessId = await this.createBusinessAA(AmalgamationTypes.HORIZONTAL)
+      const amalgamationUrl = `${this.getCreateUrl}?id=${businessId}`
+      navigate(amalgamationUrl)
+      return
+    } catch (error) {
+      console.log('Error: unable to amalgamate now =', error)
+      this.setStartingAmalgamationSpinner(false)
+      this.showErrorDialog = true
+    }
+  }
+
+  /** Called when Start Short form Vertical button is clicked. */
+  async startVerticalAmalgamation (): Promise<any> {
+    try {
+      // show spinner since this is a network call
+      this.setStartingAmalgamationSpinner(true)
+      const businessId = await this.createBusinessAA(AmalgamationTypes.VERTICAL)
+      const amalgamationUrl = `${this.getCreateUrl}?id=${businessId}`
+      navigate(amalgamationUrl)
+      return
+    } catch (error) {
+      console.log('Error: unable to amalgamate now =', error)
+      this.setStartingAmalgamationSpinner(false)
+      this.showErrorDialog = true
+    }
+  }
```

Comment:
How about combining these THREE methods and passing the amalgamation type in as a formal parameter?

Reply:
Yes.  I realized this after creating a PR